src/pages/home/report/ReportActionCompose.js

```diff
@@ -134,7 +155,21 @@ class ReportActionCompose extends React.Component {
         this.setTextInputShouldClear(true);
     }
 
+    /**
+     * Fired when the windows dimensions changes
+     * @param {Object} changedWindow
+     */
+    disableInputBasedOnDimensions({window: changedWindow}) {
+        const isSmallScreenWidth = changedWindow.width <= variables.mobileResponsiveWidthBreakpoint;
+        this.setState({
+            isSmallScreenWidth,
+        });
+    }
+
     render() {
+        const inputDisable = this.props.isSidebarShown
```

Comment:
I would add a comment here:

// We want to make sure to disable on small screens because in iOS safari the keyboard up/down buttons will focus this from the chat switcher. 
// https://github.com/Expensify/Expensify.cash/issues/1228

Reply:
Replaced comment.